servicetalk-concurrent-api-internal/src/main/java/io/servicetalk/concurrent/api/internal/BlockingUtils.java

```diff
@@ -24,13 +24,19 @@
 import static io.servicetalk.concurrent.api.Completable.fromRunnable;
 import static io.servicetalk.utils.internal.PlatformDependent.throwException;
 
-final class BlockingUtils {
+/**
+ * Common utility functions to unwrap {@link ExecutionException} from async operations.
+ */
+public final class BlockingUtils {
 
     private BlockingUtils() {
         // no instances
     }
 
-    interface RunnableCheckedException {
+    /**
+     * Wrapper {@link Runnable} that captures and converts exceptions to unchecked.
+     */
+    public interface RunnableCheckedException {
```

Comment:
We discussed offline that to avoid a custom runnable type in our internal classes, we can add `Completable.fromCallbable(Callable<Void>)` static method. Consider submitting a PR for that first, then we won't need to make this type public.

Reply:
I also wonder what methods we need from this class externally. IIRC this class pre-dates the `Future` and other blocking conversion methods on our async sources.

Reply:
@Scottmitch the interesting parts is the `blockingInvocation` APIs. The public surface is small, nevertheless is nice not to repeat this code-block in various spots.